feature/apartment/src/main/java/com/iteneum/apartment/presentation/ApartmentView.kt

```diff
@@ -0,0 +1,223 @@
+                fontSize = MaterialTheme.typography.titleSmall.lineHeight
+            )
+            LpGenericChip(label = "+${stringResource(id = R.string.newA)}") {
+                /*TODO with this button the user could make a repair request for him department*/
+            }
+        }
+        RepairStatusRow(
+            modifier = Modifier.fillMaxWidth(),
+            titleStatusRepairs = stringResource(id = R.string.open)
+        )
+        Divider(
+            thickness = LeasePertTheme.sizes.stroke,
+            color = MaterialTheme.colorScheme.onPrimary,
+            modifier = Modifier.padding(
+                start = LeasePertTheme.sizes.middleSize,
+                end = LeasePertTheme.sizes.middleSize
+            )
+        )
+        RepairStatusRow(
+            modifier = Modifier.fillMaxWidth(),
+            titleStatusRepairs = stringResource(id = R.string.in_progress)
+        )
+        Divider(
+            thickness = LeasePertTheme.sizes.stroke,
+            color = MaterialTheme.colorScheme.onPrimary,
+            modifier = Modifier.padding(
+                start = LeasePertTheme.sizes.middleSize,
+                end = LeasePertTheme.sizes.middleSize
+            )
+        )
+        RepairStatusRow(
+            modifier = Modifier.fillMaxWidth(),
+            titleStatusRepairs = stringResource(id = R.string.closed)
+        )
+    }
+}
+
+@Composable
+fun RepairStatusRow(
+    modifier: Modifier = Modifier,
+    titleStatusRepairs: String,
+    quantityStatusRepairs: Int = 0,
+) {
+    Row(
+        modifier = modifier
+            .padding(
+                start = LeasePertTheme.sizes.minorRegularSize,
+                end = LeasePertTheme.sizes.minorRegularSize
+            ), horizontalArrangement = Arrangement.SpaceBetween
+    ) {
+        Text(
+            modifier = Modifier.padding(
+                top = LeasePertTheme.sizes.smallSize,
+                bottom = LeasePertTheme.sizes.extraSize18
+            ),
+            text = titleStatusRepairs,
+            style = MaterialTheme.typography.bodyLarge
+        )
+        Box(
+            modifier = Modifier
+                .padding(
+                    bottom = LeasePertTheme.sizes.smallerSize,
+                    top = LeasePertTheme.sizes.extraSize10
+                )
+                .size(LeasePertTheme.sizes.largeSize)
+                .clip(MaterialTheme.shapes.extraLarge)
+                .background(color = MaterialTheme.colorScheme.secondaryContainer),
+            contentAlignment = Alignment.Center
+        ) {
+            Text(
+                text = quantityStatusRepairs.toString(),
+                color = MaterialTheme.colorScheme.onPrimary
+            )
+        }
+    }
+}
+
+@Composable
+fun InformationSection() {
+    Column(modifier = Modifier.padding(top = LeasePertTheme.sizes.midSmallSize)) {
+        Text(
+            text = stringResource(id = R.string.information),
```

Comment:
Please check the fontWeight for each section title

Reply:
They are applied automatic with the theme of te design system